engines/scumm/detection.cpp

```diff
@@ -243,6 +243,15 @@ static const ExtraGuiOption audioOverride {
 	0
 };
 
+static const ExtraGuiOption enableOriginalGUI{
```

Comment:
Please use more readable `enableOriginalGUI = {`, e.g. by adding the assignment operator

Reply:
Whoops, big oversight! Thanks, done!